### ISSUE TITLE
Unable to resolve spec: :meander.syntax.node.epsilon/partition

### ISSUE THREAD

**Issue by theronic** [noprompt/meander#85]
I added `[meander/epsilon "0.0.300"]` to :dependencies.

Attempting to `(require '[meander.epsilon :as m])` throws:

```
Syntax error macroexpanding r.match/search at (epsilon.cljc:21:4).
Unable to resolve spec: :meander.syntax.node.epsilon/partition
```

If I manually evaluate meander.epsilon.clj I see:

```
Syntax error compiling at (epsilon.cljc:1:1).
namespace 'meander.substitute.epsilon' not found
=> #'meander.epsilon/match
=> #'meander.epsilon/find
=> #'meander.epsilon/search
=> #'meander.epsilon/breadth-first-search
=> #'meander.epsilon/subst
Syntax error compiling at (form-init9644403811299032431.clj:110:3).
No such namespace: r
=> #'meander.epsilon/rewrite
=> meander.epsilon/rewrite
Syntax error compiling at (form-init9644403811299032431.clj:142:3).
No such namespace: r
=> #'meander.epsilon/rewrites
=> meander.epsilon/rewrites
Syntax error reading source at (REPL:171:22).
Invalid token: ::r.syntax/phase
```

**Comment by jimmyhmiller** [comment]
@theronic Really sorry that happened to you.

There was a typo in our specs. From what I can tell in your project you have `(clojure.specs.alpha/check-asserts true)`. If spec asserts are turned off, it doesn't throw any error.

I have fixed this in https://github.com/noprompt/meander/commit/fa41ed814807cef38fbdf8e1be1e2036671504bc and turned on asserts during our tests so that it doesn't happen again. 

There should be a release out soon that will have this issue fixed.

**Comment by jimmyhmiller** [comment]
Fixed in `[meander/epsilon "0.0.301"]`.

Thanks for reporting this. Let us know if you have any more issues.
